topics/homework_topic4_Lobashova/OOP.py

```diff
@@ -0,0 +1,81 @@
+# Write a TestCase class with methods for setup(), run(), and teardown().
+# Create objects of the TestCase class to represent individual test cases.
+class TestCase:
+    # Constructor of the class
+    def __init__(self, name):
+        self.name = name  # Attribute/Property/State
+
+    def setup(self):
+        print(f"Preparing for: {self.name}")
+
+    def run(self):
+        print(f"Running test case: {self.name}")
+
+    def teardown(self):
+        print(f"Clearing after: {self.name}")
+
+
+print('For first test')
+test = TestCase("Login Test")
```

Comment:
Be consistent with the quotes, even though Python accepts both. This is a good practice, not a rule.